README.md

```diff
@@ -1,15 +1,151 @@
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.typesafe.akka/akka-stream-kafka_2.11/badge.svg)](https://maven-badges.herokuapp.com/maven-central/com.typesafe.akka/akka-stream-kafka_2.11)
 If you have questions or are working on a pull request or just curious, please feel welcome to join the chat room: [![Join the chat at https://gitter.im/akka/reactive-kafka](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/akka/reactive-kafka?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
 
-[Reactive Streams](http://www.reactive-streams.org) wrapper for [Apache Kafka](https://kafka.apache.org/).  
+[Akka Streams](http://doc.akka.io/docs/akka/2.4.3/scala/stream/index.html) connector for [Apache Kafka](https://kafka.apache.org/).
 
 Created and maintained by 
 [<img src="https://softwaremill.com/img/logo2x.png" alt="SoftwareMill logo" height="25">](https://softwaremill.com)
 
-Supports Kafka 0.9.0.x  
+## New API: 0.11-M1
+
+Supports Kafka 0.9.0.x
+
+This version of `akka-stream-kafka` depends on Akka 2.4.3 and Scala 2.11.8. 
+
+Available at Maven Central for Scala 2.11:
+
+````scala
+libraryDependencies += "com.typesafe.akka" %% "akka-stream-kafka" % "0.11-M1"
+````
+
+Example usage
+----
+
+#### Scala
+
+Producer Settings:
+
+```Scala
+import akka.kafka.scaladsl._
+import org.apache.kafka.common.serialization.StringSerializer
+import org.apache.kafka.common.serialization.ByteArraySerializer
+
+  val producerSettings = ProducerSettings(system, new ByteArraySerializer, new StringSerializer)
+    .withBootstrapServers("localhost:9092")
+```
+
+Produce messages:
+
+```Scala
+  Source(1 to 10000)
+    .map(_.toString)
+    .map(elem => new ProducerRecord[Array[Byte], String]("topic1", elem))
+    .to(Producer.plainSink(producerSettings))
+```
+
+Produce messages in a flow:
+
+```Scala
+  Source(1 to 10000)
+    .map(elem => Producer.Message(new ProducerRecord[Array[Byte], String]("topic1", elem.toString), elem))
+    .via(Producer.flow(producerSettings))
+    .map { result =>
+      val record = result.message.record
+      println(s"${record.topic}/${record.partition} ${result.offset}: ${record.value} (${result.message.passThrough}")
+      result
+    }
+```
+
+Consumer Settings:
+
+```Scala
+import akka.kafka.scaladsl._
+import org.apache.kafka.common.serialization.StringDeserializer
+import org.apache.kafka.common.serialization.ByteArrayDeserializer
+import org.apache.kafka.clients.consumer.ConsumerConfig
+
+val consumerSettings = ConsumerSettings(system, new ByteArrayDeserializer, new StringDeserializer, 
+    Set("topic1"))
+  .withBootstrapServers("localhost:9092")
+  .withGroupId("group1")
+  .withProperty(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest")
+```
+
+Consume messages and store a representation, including offset, in DB:
+
+```Scala
+  db.loadOffset().foreach { fromOffset =>
+    val settings = consumerSettings
+      .withFromOffset(new TopicPartition("topic1", 1), fromOffset)
```

Comment:
yeah, feedback is what we want
here, all those settings modifiers start with `with`
might be enough with `withOffset`